test/components/views/settings/tabs/user/__snapshots__/LabsUserSettingsTab-test.tsx.snap

```diff
@@ -67,78 +67,6 @@ exports[`<SecurityUserSettingsTab /> renders settings marked as beta as beta car
       </div>
     </div>
   </div>
-  <div
```

Comment:
This shows that users won't be able to disable the default-on, making it closer to forced-on which wasn't the plan for this release